src/topology/metric_space/basic.lean

```diff
@@ -1789,6 +1832,58 @@ begin
     (continuous_on_iff_continuous_restrict.1 hf),
 end
 
+section uniform_convergence
+
+local attribute [-instance] Pi.topological_space
+local attribute [-instance] Pi.uniform_space
+local attribute [instance] uniform_convergence.topological_space
+
+/-- For a family of functions to a (pseudo) metric spaces, a convenient way to prove
+equicontinuity at a point is to show that all of the functions share a common *local* continuity
+modulus. -/
+lemma equicontinuous_at_of_continuity_modulus {ι : Type*} [topological_space β] {x₀ : β}
+  (b : β → ℝ)
+  (b_lim : tendsto b (𝓝 x₀) (𝓝 0))
+  (F : ι → β → α)
+  (H : ∀(x:β) i, dist (F i x₀) (F i x) ≤ b x) :
```

Comment:
Can you have a version where you assume this only for `x` in a  neighborhood of `x_0`?

Reply:
Sure. Do you think I should keep the global version, or should users just use `eventually_of_forall` in this case?